doc/releases/changelog-dev.md

```diff
@@ -18,7 +18,11 @@
 
 <h4>Resource-efficient Decompositions 🔎</h4>
 
-* The new graph-based decomposition system supports `QubitUnitary`.
+* The new graph-based decomposition system now supports `QubitUnitary`.
```

Comment:
Can we be a bit more specific regarding the user impact here? E.g., 

> New decomposition rules have been added to `QubitUnitary` that can be accessed with the graph-based decomposition system. <sentence on the benefit of these decomposition rules (what can the user do now that they couldn't before?)>

Wouldn't hurt to have a code example here, too :). Even if it's just printing entries in `list_decomps`, say!